Sources/Fuzzilli/FuzzIL/ProgramTypes.swift

```diff
@@ -64,9 +64,14 @@ public struct ProgramTypes: Equatable, Sequence {
         }
     }
 
-    // Get type of variable at current instruction
-    public func getType(of variable: Variable, at instrIndex: Int) -> Type {
-        return types[variable]?.last(where: { $0.index <= instrIndex })?.type ?? .unknown
+    // Get type of variable after given instruction
+    public func getType(of variable: Variable, after instrIndex: Int) -> Type {
+        if let variableTypes = types[variable] {
+            // There should always be type info after variable definition instruction
```

Comment:
Nit: in case we ever run into this, the crash message will not be very nice (i.e. "unwrapped nil"). How about a
```
assert(variableTypes[0].index <= instrIndex, "Queried type of variable before its definition")
```
?